BDArmory/Competition/RankedFreeForAllStrategy.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using BDArmory.Control;
+using BDArmory.Core;
+using UnityEngine;
+using static BDArmory.Competition.BDAScoreService;
+
+namespace BDArmory.Competition
+{
+    public class RankedFreeForAllStrategy : OrchestrationStrategy
```

Comment:
This would be used for FFA orchestration. It needs testing and integration. I have mostly focused on the waypoints up to now.